README.md

```diff
@@ -1,44 +1,140 @@
 # ObituaryHub
 
-[ObituaryHub](Heroku site url) is a Ruby on Rails application, built by students in Module 3 (of 4) at Turing School of Software and Design in Denver, CO.
+[ObituaryHub](https://obituary-hub.herokuapp.com/) is a Ruby on Rails application, built by students in Module 3 (of 4) at Turing School of Software and Design in Denver, CO.
+It was inspired by the COVID-19 pandemic and the climbing death toll in the United States, as well as the abundance of obituaries being published in honor of victims of COVID-19. Because of the pandemic, we as a society are being forced to deal with the overwhelming death toll in isolation; there are few opportunities to express our grief collectively. The obituary, already a weighty document, has become an essential way that we are able to communicate our grief and to invite others to join us in mourning as well as celebrate the lives of those we've lost.
+
+Many of us can think back to our childhood when our parents or grandparents would read the obituary section of the newspaper, discuss people in the community who had passed away, send condolences to friends and family with whom we were acquainted, and make plans to attend a funeral or memorial. Today, we often find out that friends, family, and community members have passed away on Facebook. It is traumatic to discover a loved one has died while scrolling through the Facebook newsfeed, sifting through ads, politial posts, and memes.
+
+We believe these beautiful expressions of love and sorrow deserve a better ecosystem. This is why we created [ObituaryHub](https://obituary-hub.herokuapp.com/). It is built with the dignity of the deceased in mind as well as the tender hearts of those in mourning. It is a place to celebrate life and honor death.
 
 ### Team
 
-[Stella Bonnie]()
-[Jack Cullen]()
-[Eric Larson]()
-[Sage Lee]()
+[Stella Bonnie](https://github.com/stellakunzang)
+[Jack Cullen](https://github.com/jpc20)
+[Eric Larson](https://github.com/EricLarson2020)
+[Sage Lee](https://github.com/sagemlee)
 
 ## Reflection
 
+When we originally discussed what our Minimum Viable Product would include, and what we wanted displayed on the different pages, the word "community" never came up. As we got further along with the project and started thinking about we would use it and the benefit we would derive as users, we started describing the obituaries in our database as obituaries from our community.
+
 ### Successes
 
+**[Microservice](#microservice)
+
+**[Caching](#optimization)
```

Comment:
These little # things link to other sections of the markdown!

Reply:
ooooo thats a cool trick!